natgeo/mortar
# [SYTO-731] Only underline links when hover/active/focus

https://jira.natgeo.com/browse/SYTO-731

This is a follow up ticket to make links only be underlined on hover/active/focus

To test:
Pull SYTO-731-2
Run npm test to verify tests pass
Run gulp serve
Verify kicker on http://localhost:3000/typography/
http://localhost:3000/components/podpromo/index.html
